controllers/testTypescript.ts

```diff
@@ -0,0 +1,6 @@
+export const testTypescript = async (req: any, res: any) => {
+  return res.json({
+    message: "Working perffectly fine.",
```

Comment:
Spelling mistake 🙃

Reply:
Or was it intentional😅

Reply:
Testing file will be removed before merging to develop